### ISSUE TITLE
Wording of the Rift Button

### ISSUE THREAD

**Issue by moonglum** [AsyncTI4/TI4_map_generator_bot#1864]
First of all, let me thank you for your amazing work ❤️ I‘m really happy I can TI now more regularly by playing it async. That‘s just awesome!

One thing I stumbled over was the rift button. It says “Rift some Units”. When I flew through a rift, I was really unsure if I should press the button: Does it just destroy the units? Will it roll first? It will roll first, so I would suggest rewording it: “Units flew through a rift”. This makes it clear when you should click on the button, and it removes the uncertainty if they are destroyed with no roll.

**Comment by Fingon00** [comment]
I think this got implemented
